src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -1144,6 +1144,11 @@ cover.fluid.voiding.title=Fluid Voiding Settings
 cover.fluid.voiding.advanced.title=Advanced Fluid Voiding Settings
 cover.item.voiding.title=Item Voiding Settings
 cover.item.voiding.advanced.title=Advanced Item Voiding Settings
+cover.voiding.label.disabled=Disabled
+cover.voiding.label.enabled=Enabled
+cover.voiding.tooltip=WARNING! Setting this to "Enabled" means that fluids or items WILL be voided.
```

Comment:
To make the "WARNING!" text a bit more visible:
```suggestion
cover.voiding.tooltip=§cWARNING!§7 Setting this to "Enabled" means that fluids or items WILL be voided.
```

Reply:
Even better to make it a flashing tooltip using ToolTipHelper

Reply:
resolved with https://github.com/GregTechCEu/GregTech/pull/1414/commits/98d09660c3c684356511287d69cbe986ba8d4962. Determined that TooltipHelper couldn't be used due to `createUI()` (where the lang is referenced) is called only once when UI opens.